api/cpp/include/slint_models.h

```diff
@@ -852,6 +852,10 @@ class Repeater
 
         void row_added(size_t index, size_t count) override
         {
+            if (index > data.size()) {
```

Comment:
For a moment I was thinking it should be `>=`, but then I saw the function name :)